[Bug]: Flink 1.15 quickstart  failed for   java.lang.ClassNotFoundException

### What happened?

when do quickstart use flink 1.15, we get the exception。 
Flink SQL> SELECT id, `name` FROM arctic.test_db.test_table/*+OPTIONS('streaming' = 'true')*/;
[ERROR] Could not execute SQL statement. Reason:
java.lang.ClassNotFoundException: com.netease.arctic.shaded.org.apache.flink.streaming.connectors.kafka.table.KafkaConnectorOptions

### Affects Versions

0.3.2

### What engines are you seeing the problem on?

Flink

### How to reproduce

use flink 1.15 , according to the quickstart guide.

### Relevant log output

```shell
Flink SQL> SELECT id, `name` FROM arctic.test_db.test_table/*+OPTIONS('streaming' = 'true')*/;
[ERROR] Could not execute SQL statement. Reason:
java.lang.ClassNotFoundException: com.netease.arctic.shaded.org.apache.flink.streaming.connectors.kafka.table.KafkaConnectorOptions
```


### Anything else

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

--- a/flink/v1.15/flink-runtime/pom.xml
+++ b/flink/v1.15/flink-runtime/pom.xml
@@ -179,7 +179,7 @@
                             </excludes>
                         </filter>
                         <filter>
-                            <artifact>org.apache.flink:flink-connector-kafka_${scala.binary.version}</artifact>
+                            <artifact>org.apache.flink:flink-connector-kafka</artifact>
                             <excludes>
                                 <exclude>META-INF/**</exclude>
                                 <exclude>org/apache/flink/table/descriptors/**</exclude>
@@ -214,7 +214,7 @@
                             <include>cglib:*</include>
                             <include>com.google.guava:*</include>
                             <include>asm:*</include>
-                            <include>org.apache.flink:flink-connector-kafka_${scala.binary.version}</include>
+                            <include>org.apache.flink:flink-connector-kafka</include>
                             <include>org.apache.kafka:*</include>
                         </includes>
                     </artifactSet>

